.github/workflows/ci-linux-golden-tests.yml

```diff
@@ -22,7 +22,7 @@ jobs:
         os: [ubuntu-latest]
 
     steps:
-    - uses: actions/checkout@3df4ab11eba7bda6032a0b82a6bb43b11571feac
+    - uses: actions/checkout@8ade135a41bc03ea155e62e844d188df1ea18608
```

Comment:
Nit: you could have written the version number. Let me do that in a different PR.